### DISCUSSION TITLE
Plugin/Preset files are not allowed to export objects, only functions. #6808

### DISCUSSION THREAD

**Discussion by brunolnetto** [babel/babel#15212] [Show and tell] [answered]
Hi fellows,

I write a [template-app](https://github.com/web-needle/sappio), also called a _boilerplate_. During development, I stumbled upon an issue with the title while testing my application. The testing is in the early stage, so there is only a folder on core/routes. You may reproduce it on the shell by command run `npm run test`.

**Comment by nicolo-ribaudo** [comment, marked as answer]
You are mixing Babel 6 and Babel 7, please make sure to only use version 7: https://babeljs.io/docs/en/v7-migration
